File: haul3/data/langs/pas/libs/hio.pas
{ HAUL IO }
Unit hio;

Interface

Procedure put(data:String);
Procedure put_direct(data:String);
Procedure shout(data:String);
Function fetch():String;

Function int_str(i:Integer):String;
{Function str(f:Float):String; overload;}

Implementation

Procedure put(data:String);
Begin
	WriteLn(data);
End;
Procedure put_direct(data:String);
Begin
	Write(data);
End;
Procedure shout(data:String);
Begin
	WriteLn('!! ', data);
	ReadLn;
End;

Function fetch():String;
begin
	Result := ReadLn();
end;


Function int_str(i:Integer):String;
Var
	s:String;
begin
	{str := IntToStr(i);}
	Str(i, s);
	int_str := s;
End;
{
Function str(f:Float):String;
begin
	Result := FloatToStr(v);
End;
}
End.

File: haul3/data/langs/py/hio.py
# HAUL IO

#import sys

def put(data):
	#sys.stdout.write(data)
	print(data)

def put_direct(data):
	#sys.stdout.write(data)
	print(data),

def shout(data):
	print('!! ' + str(data))

def get():
	# PY3
	#return input()
	
	data = raw_input()
	return data

def int_str(i):
	return str(i)